### DISCUSSION TITLE
Support for Multi Tenant App

### DISCUSSION THREAD

**Discussion by ryanbaldisco** [santigarcor/laratrust#683] [Q&A]
@santigarcor Great package. Totally impressed. But I have to ask though, does this package supports multi-tenant setup laravel application? If it doesn't, is there a way to manually implement it using this package? Maybe similar with Bouncer https://github.com/JosephSilber/bouncer?tab=readme-ov-file#multi-tenancy . TIA. Cheers!

**Comment by santigarcor** [comment]
Do you want to implement new roles per tenant? or how do you want to deal with it?
